bonfire/bonfire.py

```diff
@@ -463,6 +496,30 @@ def _validate_resource_arguments(ctx, param, value):
     ),
 ]
 
+_reservation_process_options = [
+    click.option(
+        "--name",
+        "-n",
```

Comment:
You can just remove the short arg here entirely, and require people to type out `--name` fully -- I guess the only question is -- do we expect it is going to be less common to extend a reservation by its actual name vs ns?

If we expect it to be used a lot maybe a short arg is needed... otherwise I'd say only a long arg is fine.

Reply:
I'm not sure. I'm fine switching to just the long arg for name though.